code/modules/mining/equipment/mining_tools.dm

```diff
@@ -143,6 +145,9 @@
 	effectiveness = 40, \
 	)
 	//it's sharp, so it works, but barely.
+	AddElement(/datum/element/gravedigger, \
+	dig_time = grave_speed, \
```

Comment:
```suggestion
		dig_time = grave_speed, \
```

Reply:
What? I thought I got rid of this arg entirely in factor of just using use_tool